[14-Mar-2021 22:12:01 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:05 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:05 Europe/Berlin] Uncaught exception: 'Exception' with message ' No route matched'
Stack trace: #0 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown inC:\xampp\htdocs\feedback\Core\Router.php on line 83
[14-Mar-2021 22:12:11 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:30 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:30 Europe/Berlin] Uncaught exception: 'Exception' with message ' No route matched'
Stack trace: #0 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown inC:\xampp\htdocs\feedback\Core\Router.php on line 83
[14-Mar-2021 22:12:36 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:41 Europe/Berlin] Uncaught exception: 'Exception' with message ' Wrong login, please try again'
Stack trace: #0 [internal function]: App\Controllers\Admin\Users->loginAction()
#1 C:\xampp\htdocs\feedback\Core\Controller.php(22): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\feedback\Core\Router.php(72): Core\Controller->__call('login', Array)
#3 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('admin/users/log...')
#4 {main}
Thrown inC:\xampp\htdocs\feedback\App\Controllers\Admin\Users.php on line 25
[14-Mar-2021 22:12:42 Europe/Berlin] Uncaught exception: 'Exception' with message ' No route matched'
Stack trace: #0 C:\xampp\htdocs\feedback\public\index.php(20): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown inC:\xampp\htdocs\feedback\Core\Router.php on line 83
